cmake/GitUpdateSubmodules.cmake

```diff
@@ -2,6 +2,13 @@ find_package(Git QUIET)
 
 if(EXISTS "${NGEN_ROOT_DIR}/.git")
```

Comment:
Not relevant to this PR, and really an edge case in general, but I think this condition may not work in the case of a git 'worktree', which just leaves some sort of link or reference in the working source directory to the actual repository.